lib/pharos/scripts/configure-etcd.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -e
+
+if [ -e /etc/kubernetes/manifests/pharos-etcd.yaml ]; then
+  exit
+fi
+
+mkdir -p /etc/systemd/system/kubelet.service.d
+cat <<EOF >/etc/systemd/system/kubelet.service.d/5-pharos-etcd.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/kubelet --pod-manifest-path=/etc/kubernetes/manifests/ --read-only-port=0 --cadvisor-port=0
```

Comment:
@SpComb should we lock-down this more?

Reply:
I don't entirely understand what's going on here... are we not using the `10-kubeadm.conf` dropin onto these nodes?

This configuration is really unsafe, AFAIK it leaves the kubelet API wide open and unauthenticated.